packages/yoga/src/Snackbar/web/Snackbar.test.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import { ThemeProvider, Snackbar } from '../..';
```

Comment:
This is pretty ugly =D

Reply:
This comes from a scaffold template. All components have this pattern.